src/cyclotom.c

```diff
@@ -149,12 +149,12 @@ void GrowResultCyc(UInt size) {
     UInt i;
     if (STATE(ResultCyc) == 0) {
         STATE(ResultCyc) = NEW_PLIST( T_PLIST, size );
-        res = &(ELM_PLIST( STATE(ResultCyc), 1 ));
+        res = BASE_PTR_PLIST(STATE(ResultCyc));
```

Comment:
Commit message of the commit introducing this line is bad: "Some pieces of code were abuse the" -> "abusing" perhaps?